nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -391,6 +391,38 @@ let
   );
 
   mkCertOwnershipAssertion = import ../../../security/acme/mk-cert-ownership-assertion.nix;
+
+  snakeOilCert = pkgs.runCommand "nginx-config-validate-cert" { nativeBuildInputs = [ pkgs.openssl.bin ]; } ''
```

Comment:
Not sure if we want to use runCommandLocal here. We probably don't want to cache or substitute this, or do we?

Reply:
this certificate is only used during build, so reusing it or sharing it via a binary cache is not a problem